etl/steps/data/garden/climate/2024-11-19/total_precipitation.py

```diff
@@ -0,0 +1,60 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+import calendar
+
+import owid.catalog.processing as pr
+
+from etl.data_helpers import geo
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load meadow dataset and read its main table.
+    ds_meadow = paths.load_dataset("total_precipitation")
+    tb = ds_meadow["total_precipitation"].reset_index()
+
+    #
+    # Process data.
+    #
+    # Harmonize country names.
+    tb = geo.harmonize_countries(
+        df=tb, countries_file=paths.country_mapping_path, excluded_countries_file=paths.excluded_countries_path
+    )
+    # Extract year and month as integers
+    tb["year"] = tb["time"].astype(str).str[0:4].astype(int)
+    tb["month"] = tb["time"].astype(str).str[5:7].astype(int)
+
+    # Get the number of days in the given month and year
+    tb["days_in_month"] = tb.apply(lambda row: calendar.monthrange(row["year"], row["month"])[1], axis=1)
+
+    # Use the number of days to convert to monthly totals rather than daily averages - as per info here https://confluence.ecmwf.int/pages/viewpage.action?pageId=197702790
+    tb["total_precipitation"] = tb["total_precipitation"] * 1000 * tb["days_in_month"]
+
+    # Use the baseline from the Copernicus Climate Service https://climate.copernicus.eu/surface-air-temperature-january-2024
+    tb_baseline = tb[(tb["year"].astype(int) > 1990) & (tb["year"].astype(int) < 2021)]
```

Comment:
Shouldn't it be >= and <=?

Reply:
nope actually it's 1991-2020. Not sure why this is the standard [baseline period for temperature anomalies](https://climate.copernicus.eu/copernicus-2024-virtually-certain-be-warmest-year-and-first-year-above-15degc#:~:text=The%20average%20global%20temperature%20anomaly,the%20warmest%20year%20on%20record.) so I am just reusing it here

Reply:
Ah, my bad. Thanks!